usng4j-impl/src/main/java/org/codice/usng4j/impl/CoordinateSystemTranslatorImpl.java

```diff
@@ -288,23 +288,18 @@ public UtmCoordinate toUtm(final DecimalDegreesCoordinate decimalDegreesCoordina
                         + (61 - 58 * T + T * T + 600 * C - 330 * this.eccPrimeSquared)
                             * (A * A * A * A * A * A)
                             / 720)));
+    UTMNorthing += (UTMNorthing < 0) ? NORTHING_OFFSET : 0;
 
     return new UtmCoordinateImpl(zoneNumber, UTMEasting, UTMNorthing);
   }
 
   @Override
   public UpsCoordinate toUps(final DecimalDegreesCoordinate decimalDegreesCoordinate) {
-    final UtmUpsCoordinate utmUpsCoordinate = toUtmUps(decimalDegreesCoordinate);
-    if (utmUpsCoordinate.isUTM()) {
+    if (!isInUPSSpace(decimalDegreesCoordinate)) {
```

Comment:
❓ Why was this put in instead of `isUTM`?

Reply:
❓ Isn't there some overlap between UTM and UPS space?